client/flutter/.vscode/tasks.json

```diff
@@ -0,0 +1,17 @@
+{
+    // See https://go.microsoft.com/fwlink/?LinkId=733558
```

Comment:
What is this file for?

Reply:
Adds a VS Code build task that can be run instead of running `flutter pub run flutter_native_splash:create`. I had updated the PR description under "What does this PR accomplish?" to describe it.